From 1cd3fe4083eb8784fa2a125e4af0edb80e6928b5 Mon Sep 17 00:00:00 2001
From: Sven Eden <sven.eden@prydeworx.com>
Date: Thu, 12 Mar 2020 20:31:00 +0100
Subject: [PATCH] basic/selinux-util.c : Uncomment missing includes (#157)

Compiling on Debian unstable failed using gcc 9.2.1 and libc6 2.29.
Uncommenting the two commented includes in basic/selinux-util.c
fixes the compilation issues.

Bug: #157
Closes: #157
Signed-off-by: Sven Eden <sven.eden@prydeworx.com>
---
 src/basic/selinux-util.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/basic/selinux-util.c b/src/basic/selinux-util.c
index 40a85feec..3be9f2c0d 100644
--- a/src/basic/selinux-util.c
+++ b/src/basic/selinux-util.c
@@ -1,13 +1,13 @@
 /* SPDX-License-Identifier: LGPL-2.1+ */
 
 #include <errno.h>
-//#include <fcntl.h>
+#include <fcntl.h>
 #include <malloc.h>
 #include <stddef.h>
 #include <string.h>
 #include <sys/stat.h>
 #include <sys/time.h>
-//#include <sys/types.h>
+#include <sys/types.h>
 #include <sys/un.h>
 #include <syslog.h>
 
